packages/esm-service-queues-app/src/create-queue-entry/existing-visit-form/existing-visit-form.component.tsx

```diff
@@ -22,22 +22,26 @@ const ExistingVisitForm: React.FC<ExistingVisitFormProps> = ({ visit, closeWorks
   const [isSubmitting, setIsSubmitting] = useState(false);
 
   const { mutateQueueEntries } = useMutateQueueEntries();
-  const [submitQueueEntry, setSubmitQueueEntry] = useState<(visit: Visit, patientUuid: string) => Promise<any>>(null);
+  const [callback, setCallback] = useState<{
+    submitQueueEntry: (visit: Visit) => Promise<any>;
+  }>(null);
 
   const handleSubmit = useCallback(
     (event) => {
       event.preventDefault();
       setIsSubmitting(true);
 
-      submitQueueEntry?.(visit, visit.patient.uuid)
+      callback
+        .submitQueueEntry?.(visit)
```

Comment:
Isn't is possible (if unlikely) for `callback` still be `null` here?

Reply:
yeah... will change.